### DISCUSSION TITLE
Support <PackageIcon> in SeqApp packages

### DISCUSSION THREAD

**Discussion by CrossBound** [datalust/seq-tickets#1093] [Ideas]
The nuget specification allows you to embed a project image into your nuget package. It looks like Seq doesn't respect this though and still just gives you a default image for your custom app. 

It would be nice if Seq would support our custom app images instead of only being able to use the default.

**Comment by CrossBound** [comment]
in my csproj file I use the 
```xml
<PackageIcon>content/LeafAcorn_100_web.ico</PackageIcon>
```
and
```xml
<ItemGroup>
   <!-- irrelevant items omitted -->
  <None Include="LeafAcorn_100_web.ico" Pack="true" PackagePath="content" />
</ItemGroup>
```

**Comment by nblumhardt** [comment]
Thanks for the suggestion 👍

The current packaging implementation does support `PackageIconUrl` (you can see an example at https://github.com/datalust/seq-input-healthcheck/blob/5e34fb6583e07902eb5cba3a0e946360e61b8153/src/Seq.Input.HealthCheck/Seq.Input.HealthCheck.csproj#L9 ) - though the recent push from NuGet has been towards embedded images.

**Comment by CrossBound** [comment]
I believe I tried PackageIconUrl also and it also did not work, but I'll double check that.

**Comment by nblumhardt** [comment]
:+1: thanks for the follow-up.

![image](https://user-images.githubusercontent.com/342712/76259637-c6a23f80-62a1-11ea-8c66-59b0c41f8de9.png)

This one is: https://github.com/datalust/seq-apps/blob/master/src/Seq.App.EmailPlus/Seq.App.EmailPlus.nuspec#L12 (using the old NuSpec format, but setting the same field).

**Comment by vlm---** [comment]
Using IconUrl from a package is not very useful in offline environments, but not a big deal.

**Comment by nblumhardt** [comment]
Thanks @vlm--- - yes, definitely some good reasons to support the new mechanism 👍
